### DISCUSSION TITLE
Octal Literals

### DISCUSSION THREAD

**Discussion by lavezza** [stevedekorte/io#462] [Ideas]
Hi. I've been interested in Io for years but never had time to dig into it until now. I know the language isn't actively developed anymore but was wondering if pull requests for new features would still be accepted. 

As a specific example, I have code that adds support for Octal literals (0o123 == 83). It's mostly copy/paste of the current hex number literal code. 

I think support was planned since there was already a IoSeq_rawAsDoubleFromOctal function in IoSeq.c, but it needed to be fleshed out and support added to the lexer.

I'm probably going to add binary literal support to my local copy (0b1100 == 12) as an exercise. 

If the maintainers are interested I can create issues and do some pull requests.

Pat

**Comment by stevedekorte** [comment]
Hi, yes - patches are still accepted.
